services/http/api/lib/models/source.ts

```diff
@@ -0,0 +1,14 @@
+import { ModelRelatedNodesI, NeogmaInstance, NeogmaModel } from "neogma";
+import { UserMeasurementInstance, UserMeasurementModel } from "./measurement";
+
+export type SourceProperties = {
+    type: string
```

Comment:
Should be 'name' not type.

Reply:
Already changed with the merged qol-survey branch. Resolving.